src/vs/workbench/contrib/debug/common/debugModel.ts

```diff
@@ -1075,14 +1078,36 @@ export class ExceptionBreakpoint extends BaseBreakpoint implements IExceptionBre
 		result.enabled = this.enabled;
 		result.supportsCondition = this.supportsCondition;
 		result.condition = this.condition;
+		result.fallback = this.fallback;
 
 		return result;
 	}
 
+	setSupportedSession(sessionId: string, supported: boolean): void {
+		if (supported) {
+			this.supportedSessions.add(sessionId);
+		}
+		else {
+			this.supportedSessions.delete(sessionId);
+		}
+	}
+
+	setFallback(isFallback: boolean) {
+		this.fallback = isFallback;
```

Comment:
Seems like there should be a nicer way to accomplish this. Seems like `fallback` is not a concept that an individual exception breakpoint should be concerned with. But I'm not sure what the solution is, and I am ok with keeping it as-is. If this is kept, please add a comment describing what `fallback` means, it wasn't immediately clear.

Reply:
I could not find a better way, especially which allows for easily persisting this as well. I've added some comments, which should help understanding the need for this.